### PR TITLE
add dovetail 5

### PR DESCRIPTION
**Pull request by zkamvar**
This adds dovetail number 5, announcing that the beta phase is put on hold.

### CODE REVIEW

File: _posts/2022/07/2022-07-20-dovetail-005.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+## Announcements
+
+### Workbench Beta Phase On Hold
+
+**The Workbench Beta phase is [on Hold until after CarpentryCon](https://github.com/carpentries/workbench/discussions/22)**, 
+and it is likely that it will be on hold until mid-September. Over the last few
+weeks, [like an unexpected late spring ice storm](https://www.weather.gov/gjt/Jan09_IceStorm),
+situations unrelated to development of The Workbench both personal and
+professional ([such as the departure of a good friend and colleague]({{ site.url }}/blog/2022/07/thank-you-francois/)) sources
+have arisen, which forced me to stop pedaling and gently squeeze the brakes on
```
Comment (tobyhodges) [inline _posts/2022/07/2022-07-20-dovetail-005.md]:
```suggestion
situations unrelated to development of The Workbench both personal and
professional ([such as the departure of a good friend and colleague]({{ site.url }}/blog/2022/07/thank-you-francois/))
have arisen, which forced me to stop pedaling and gently squeeze the brakes on
```

I think this reads better?

File: _posts/2022/07/2022-07-20-dovetail-005.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+Toby Hodges and myself. Check out the times:
+
+ - 10 August, 2022 09:30-11:30 UTC [(add to your calendar)](https://www.google.com/calendar/event?eid=N2xtMG8ydXUwOGM5amptY3FqbjBsMWFldTkgY190bXRya2YzMjhnanRjczF1MDVtaGZibjdxY0Bn)
+ - 10 August, 2022 22:00-00:00 UTC [(add to your calendar)](https://www.google.com/calendar/event?eid=NGpmZnVmamJ1MGFqMTRzdjJsYTZsODA3YmggY190bXRya2YzMjhnanRjczF1MDVtaGZibjdxY0Bn)
+ - 12 August, 2022 14:00-16:00 UTC [(add to your calendar)](https://www.google.com/calendar/event?eid=N2lvOXM4c2F2MmJpOG4yODlmZ2hlNDliYmYgY190bXRya2YzMjhnanRjczF1MDVtaGZibjdxY0Bn)
+
+#### RSECon2022 (September 2022)
+
+Toby Hodges and I will be leading a short workshop at The Research Software
+Engineer Conference entitled "Collaborative Lesson Development with The
```
Comment (tobyhodges) [inline _posts/2022/07/2022-07-20-dovetail-005.md]:
```suggestion
Toby Hodges and I will be leading a short workshop at 
[The Conference for Research Software Engineering](https://rsecon2022.society-rse.org/) 
entitled "Collaborative Lesson Development with The
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Suggesting a few small changes, but otherwise it looks good. I am grateful to you for taking the decision to put the Workbench Beta on hold @zkamvar 🙌

File: _posts/2022/07/2022-07-20-dovetail-005.md (hunk 2 above)
Comment (zkamvar) [reply _posts/2022/07/2022-07-20-dovetail-005.md]: Thank you! I was going to include this and then forgot!

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**